app/models/symptom.rb

```diff
@@ -24,15 +24,32 @@ def self.valid_types
     where(['(name = ? OR name = ?) AND bool_value = ?', 'fever', 'used-a-fever-reducer', true])
   }
 
-  def bool_based_prompt(lang = :eng)
+  def bool_based_prompt(lang = :eng, simplified: false)
     I18n.backend.send(:init_translations) unless I18n.backend.initialized?
     case type
     when 'BoolSymptom'
-      I18n.t("assessments.symptoms.#{name}.name", locale: lang)
+      if simplified
+        I18n.t("assessments.symptoms.#{name}.name-simplified", locale: lang, default: I18n.t("assessments.symptoms.#{name}.name", locale: lang))
+      else
+        I18n.t("assessments.symptoms.#{name}.name", locale: lang)
+      end
     when 'IntegerSymptom', 'FloatSymptom'
       [
-        I18n.t("assessments.symptoms.#{name}.name", locale: lang),
-        I18n.t("assessments.threshold-op.#{threshold_operator.parameterize}", locale: lang), value
+        if simplified
+          I18n.t("assessments.symptoms.#{name}.name-simplified", locale: lang, default: I18n.t("assessments.symptoms.#{name}.name", locale: lang))
+        else
+          I18n.t("assessments.symptoms.#{name}.name", locale: lang)
+        end,
```

Comment:
Nitpick: since this if-else is in this method twice, you could just do it once before the `case` statement and store it in a variable.

Reply:
https://github.com/SaraAlert/SaraAlert/pull/1171/commits/f1b471ff40f86d955fba92e793cabe7a2e3e58b2#diff-65127faed43c03b0023573466f774d8ae85ea7dfadaf7791cbc900ecd4ff2c11R29